etna/search/views.py

```diff
@@ -696,6 +706,8 @@ class WebsiteSearchView(BucketsMixin, BaseFilteredSearchView):
     form_class = WebsiteSearchForm
     template_name = "search/website_search.html"
     search_tab = SearchTabs.WEBSITE.value
+    page_type = "Website search page"
+    page_title = "Website search"
```

Comment:
If you've checked Website search page, it now using `NativeWebsiteSearchView`, move it there now to implement.